manager/apis/app/v1alpha1/plotter_types.go

```diff
@@ -35,11 +320,20 @@ type PlotterStatus struct {
 	// +optional
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
 
+	// +required
+	Flows []FlowStatus `json:"flows"`
+
+	// Assets is a map containing the status per asset.
+	// The key of this map is assetId
+	// +required
+	Assets map[string]AssetStatus `json:"assets"`
+
 	// + optional
 	Blueprints map[string]MetaBlueprint `json:"blueprints,omitempty"`
```

Comment:
This is the blueprint status?

Reply:
Yes, in addition to the per asset status the original blueprint status is left unchanged.